tasks/main.yml

```diff
@@ -10,6 +10,15 @@
     name: td-agent
     state: "{{ fluentd_package_state }}"
 
+- name: "Add user to {{ fluentd_groups | join(',') }} group"
+  user:
+    name: td-agent
+    group: td-agent
```

Comment:
Not sure if it is mandatory?

Reply:
I mean specifying `group: td-agent`

Reply:
I don't think it's mandatory, but for informations about primary group